change/@ni-nimble-components-77505764-9cc9-4b24-924f-d89305f62204.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Don't forget to update the component status table too (not a huge deal if you wait to the very end, but somewhat nicer to have it always up to date).

Reply:
Will do this with the Angular/Blazor PR since I plan to have it up by tomorrow.